lib/index.d.ts

```diff
@@ -1458,9 +1460,9 @@ declare namespace Stardog {
          *
          * @param {Connection} conn the Stardog server connection
          * @param {string} name the data source name
-         * @param {string} dataSourceQuery the data source query
+         * @param {string | DataSourceQuery} dataSourceQuery the data source query
```

Comment:
it would be good to denote here somewhere that `DataSourceQuery` is only available on 8.2+

also, can you update the docs via `npm run docs` as well?